Platforms/QemuQ35Pkg/QemuFlashFvbServicesRuntimeDxe/FwBlockServiceDxe.c

```diff
@@ -224,26 +224,9 @@ SetPcdFlashNvStorageBaseAddresses (
   VOID
   )
 {
-  RETURN_STATUS  PcdStatus;
-
   //
-  // Set several PCD values to point to flash
+  // Do nothing.

```

Comment:
nit: Do we still keep the difference between our version and ovmf with `MU_CHANGE`?

Reply:
I'm not sure I see a point, I mean we have officially forked Q35, right? at this point we've done a pretty major refactor without using those tags so rebasing is out the window anyways.